sky/engine/core/css/CSSFontSelector.cpp

```diff
@@ -43,14 +43,13 @@ namespace blink {
 
 CSSFontSelector::CSSFontSelector(Document* document)
     : m_document(document)
-    , m_genericFontFamilySettings(document->frame()->settings()->genericFontFamilySettings())
 {
+    if (m_document)
+        m_genericFontFamilySettings = document->frame()->settings()->genericFontFamilySettings();
```

Comment:
We can actually just delete this line of code if you prefer.  There's no way to modify the genericFontFamilySettings in Settings today anyway.